app/src/main/java/com/simpragma/recipe/pojoclass/RecipeList.java

```diff
@@ -0,0 +1,87 @@
+
+package com.simpragma.recipe.pojoclass;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class RecipeList {
```

Comment:
As mentioned above, please rename this class

Reply:
Change it as RecipePojoClass